packages/editor/example/app-bar/dialogs/ModelSchemaDialog.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'recompose';
+import { StyleRulesCallback, withStyles, WithStyles } from '@material-ui/core/styles';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import Dialog from '@material-ui/core/Dialog';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import withMobileDialog from '@material-ui/core/withMobileDialog';
+import { getData, getUiSchema } from '@jsonforms/core';
+
+const styles: StyleRulesCallback<'textarea'> = () => ({
+  textarea: {
+    width: 400,
+    height: 600,
+    whiteSpace: 'pre-wrap',
+    overflowWrap: 'normal',
+    overflowX: 'scroll'
+  }
+});
+
+interface ModelSchemaDialogProps {
+  onClose: any;
+  readOnly: boolean;
+  open: boolean;
+  fullScreen?: boolean;
+  rootData?: any;
+}
+
+class ModelSchemaDialog extends
+  React.Component<ModelSchemaDialogProps & WithStyles<'textarea'>, {}> {
+  private textInput;
```

Comment:
Use `React.createRef` the ref

Reply:
We might need to update material-ui in material workspace as well. https://github.com/mui-org/material-ui/issues/11709